### ISSUE TITLE
question about editing mesh

### ISSUE THREAD

**Issue by guayabas** [PixarAnimationStudios/OpenSubdiv#365]
Hi,
Currently I am working in a project where we need to change a triangle once an interaction with it occurs, for example, if a ray cross the triangle subdivide it ("easy" since OpenSubdiv has already all that in place) but what about a reverse operation like removing the entire triangle, is it possible to achieve that?

After reading a little bit the tutorials and documentation (http://graphics.pixar.com/opensubdiv/docs/subdivision_surfaces.html#introduction) I was thinking should be "easy" to use the editing operations (remove vertex, edge, make hole) provided by the library to achieve such, will be possible to ask a small example of such.

In summary, can I simplify the mesh into something that is not the input mesh?

Thanks a lot =)

**Comment by manuelk** [comment]
No follow up in a while - closing this for now.
